master.cfg

```diff
@@ -152,7 +152,12 @@ def dpkgDeb():
     dpkg-scansources . /dev/null | gzip -9c > Sources.gz
     cd ..
     find debs -type f -exec sha256sum {} \; | sort > sha256sums.txt
-""")], doStepIf=lambda step: hasFiles(step) and savePackage(step))
+""")],
+            doStepIf=lambda step: hasFiles(step) and savePackage(step),
+            descriptionDone=util.Interpolate("""
+            Repository available with: apt-add-respository "deb [trusted=yes] https://ci.mariadb.org/%(prop:tarbuildnum)s/%(prop:buildername)s/debs ./"
```

Comment:
Spoke too fast, can you suggest a dump echo instead? `apt-add-respository` is not installed by default (needs software-properties-common that comes with lot's of dependencies and not bring any value).

```console
echo "deb [trusted=yes] https://ci.mariadb.org/%(prop:tarbuildnum)s/%(prop:buildername)s/debs ./" >>/etc/apt/sources.list
```

Reply:
probably pulls python.